src/main/java/com/amazon/opendistroforelasticsearch/ad/AnomalyDetectorPlugin.java

```diff
@@ -211,6 +212,11 @@ public AnomalyDetectorPlugin() {}
             anomalyDetectionIndices
         );
         RestSearchAnomalyDetectorAction searchAnomalyDetectorAction = new RestSearchAnomalyDetectorAction();
+        RestValidateAnomalyDetectorAction restValidateAnomalyDetectorAction = new RestValidateAnomalyDetectorAction(
```

Comment:
just name it to `validateAnomalyDetectorAction` to keep it consistent with other actions like `searchAnomalyDetectorAction`

Reply:
Seems the existing code is inconsistent in naming convention. I would prefer using `validateAnomalyDetectorAction` as there are more `actions` using such naming convention.

Reply:
Changed to validateAnomalyDetectorAction